autozone-poc/src/index.html

```diff
@@ -21,8 +21,16 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Autozone POC</title>
-  </head>
-  <body>
-    <div id="container"></div>
-  </body>
+
+    <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+    
+    <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
+    <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
```

Comment:
remove this

Reply:
this is needed for jquery range slider component. so cant remove it.

Reply:
install jquery like "npm i jquery"  and then you can include in module like "import $ from jquery" it will work.